def/configmap/main.go

```diff
@@ -56,17 +56,16 @@ func (mkv *multiKeyValue) String() string {
 }
 
 func (mkv *multiKeyValue) Set(v string) error {
+	if mkv == nil {
+		return fmt.Errorf("multiKeyValue is nil")
+	}
+
 	p := strings.SplitN(v, "=", 2)
 	if len(p) != 2 {
 		return fmt.Errorf("%s does not match label=value", v)
 	}
-	if mkv == nil {
-		mkv = &multiKeyValue{
-			p[0]: p[1],
-		}
```

Comment:
This doesn't work. You can't assign a value to a pointer receiver and expect the original value pointed at to be updated.